src/Store/StoreThread.re

```diff
@@ -199,6 +210,8 @@ let start =
     // TODO: Wire this up properly
     let quickmenuSubs = quickmenuSubscriptionsUpdater(newState);
```

Comment:
As we add more of these, we might want to add first-class support subscription like in TEA